protocol/v2/ssv/runner/proposer.go

```diff
@@ -189,9 +193,15 @@ func (r *ProposerRunner) ProcessConsensus(ctx context.Context, logger *zap.Logge
 		}
 	}
 
+	duty := r.BaseRunner.State.StartingDuty.(*spectypes.ValidatorDuty)
+	if r.doppelgangerHandler.ValidatorStatus(duty.ValidatorIndex) == doppelganger.SigningDisabled {
```

Comment:
it's safer to check `!= SigningEnabled` instead because there's a 3rd variant (unknown) and also in case other variants are added to the enum in the future

although since there's only really 2 possible options (can sign / can't sign), i'd switch the state or at least the method to a boolean to prevent such mistakes by callers